### DISCUSSION TITLE
Musescore crashing upon doing literally anything

### DISCUSSION THREAD

**Discussion by PentagonDark0505** [musescore/MuseScore#25316] [Q&A]
Hello there!
I'm encountering an issue where musescore just closes either upon startup or when opening/creating a new score. Also it closes when pressing any button except for M1. I also can't get into any settings, because then Musescore also closes itself. What I have noticed is that if it stays up for like 10 seconds, there is a small white square in the middle of the screen. If I try to click on it, Musescore also closes. 
I am using Musescore 4.4.3, have tried to reinstall it, also put it to factory settings. I have an intel i7 11700F, 16 GB RAM and an RTX 4070 super.

Is there anything I can do to stop Musescore from closing?

**Comment by MarcSabatella** [comment]
What is the “M1” button, and what happens when you press it?  Did a previous version work?  And do you have third party plugins (either code plugins or VST) installed?

I would also recommend asking for help on the official support forum at MuseScore.org.  GitHub here is for reproducible bug reports.  Crashes are bugs, of course, but until you can provide steps for others to reproduce the same crash you are seeing, the support forum is more appropriate.

**Comment by PentagonDark0505** [comment]
Hello there,

thanks for your answer, I meant the left mouse button with M1. If I press the left mouse button, Musescore instantly closes, but taht actually happens for any other button too. Musescore worked like 2 weeks ago, so I guess it worked with 4.4.2
I have no third party plugins.

Thank you for the information, I'll turn to the support now!
________________________________
Von: Marc Sabatella ***@***.***>
Gesendet: Samstag, 26. Oktober 2024 00:47
An: musescore/MuseScore ***@***.***>
Cc: PentagonDark0505 ***@***.***>; Author ***@***.***>
Betreff: Re: [musescore/MuseScore] Musescore crashing upon doing literally anything (Discussion #25316)


What is the “M1” button, and what happens when you press it? Did a previous version work? And do you have third party plugins (either code plugins or VST) installed?

I would also recommend asking for help on the official support forum at MuseScore.org. GitHub here is for reproducible bug reports. Crashes are bugs, of course, but until you can provide steps for others to reproduce the same crash you are seeing, the support forum is more appropriate.

**Comment by MarcSabatella** [comment]
One thing you can do here is go to the Diagnostic menu and collect your log files and post them here.
